opentelemetry-sdk/Cargo.toml

```diff
@@ -51,6 +51,7 @@ testing = ["opentelemetry/testing", "trace", "metrics", "logs", "rt-async-std",
 rt-tokio = ["tokio", "tokio-stream"]
 rt-tokio-current-thread = ["tokio", "tokio-stream"]
 rt-async-std = ["async-std"]
+no-preallocate-attributes-array = []
```

Comment:
Alternate suggestion:
`disable_stack_log_attributes`

"Preallocate" is potentially misleading, we can pre-allocate in heap as well.

Reply:
please also add a changelog entry for this. There is a general lacking of documentaion for each of the feature flags, but that is something we need to work on independent of this PR.